Scripts/application.py

```diff
@@ -4,17 +4,19 @@
 from flask import Flask
 app = Flask(__name__)
 
+path_to_db = '../db/allYourSantaAreBelongToUs.db'
+
 @app.route('/')
 def hello_world():
     return 'Hello World!'
     
 @app.route('/api/v1/person')
 def return_people():
     # returns a json of all people
-    with sqlite3.connect('../db/allYourSantaAreBelongToUs.db') as connection:
+    with sqlite3.connect(path_to_db) as connection:
         cursor = connection.cursor()
         #get all of the people
-        cursor.execute('SELECT * FROM people')
+        cursor.execute('SELECT person_ID, email, givingTo_ID FROM people')
         raw_data: List = cursor.fetchall()
         data: Dict = {}
         for person in raw_data:

```

Comment:
I think it's better to return this as a list of objects rather than a single object. So instead of
`
{
  1: {
    "email":"foo",
    ""assignment_ID": 2
  },
  2: {
    "email":"bar",
    ""assignment_ID": 1    
  }
}
`
replace with
`
[
{
  "id": 1
  "email":"foo",
  ""assignment_ID": 2
},
{
  "id": 2
  "email":"bar",
  ""assignment_ID": 1
}
]
`
I believe this is more standard with REST

Reply:
Done.